packages/experiments/src/utilities/chicletHelper.tsx

```diff
@@ -74,11 +74,13 @@ export function generateCommonHTML(
   props: IChicletCardProps,
   imageProvided: boolean,
   icon?: string,
-  preview?: string
```

Comment:
Since this function returns the preview element, should we rename it to something more inline with that?

Reply:
Can we also add TSDoc comments to these functions? The descriptions can be short